man/systemd-mount.xml

```diff
@@ -159,6 +159,14 @@
         <listitem><para>Additional mount options for the mount point.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--owner=<replaceable>USER</replaceable></option></term>
+
+        <listitem><para>Let the specified user <replaceable>USER</replaceable> own the mounted directory.
+        This is done by appending <option>uid=</option> and <option>gid=</option> options to the list
+        of mount options.</para></listitem>
+      </varlistentry>
```

Comment:
It doesn't let the user own the mounted directory; it lets them own the whole mounted _filesystem_.

Also this doc looks like it should work e.g. for ext4, but I don't believe it does.

> The mount options uid= and gid= are not valid for all available filesystems. The ones that support proper permissions usually don’t accept them, as is the case for [ext4].

Reply:
Indeed, this phrasing is wrong. I'll fix it when updating.